src/core/sys/posix/fcntl.d

```diff
@@ -159,6 +159,7 @@ version (CRuntime_Glibc)
 
         enum O_APPEND       = 0x400;    // octal    02000
         enum O_NONBLOCK     = 0x800;    // octal    04000
+        enum O_CLOEXEC      = 0x80000;  // octal 02000000
```

Comment:
`0x200000` on HPPA, `0x400000` on SPARC.

Reply:
This is in a version X86_Any block and the commit message says specifically for x86